Revit_Core_Engine/Query/SpatialBoundCondition.cs

```diff
@@ -51,13 +51,23 @@ public static partial class Query
                 }
                 else if (elem.GetBoundarySegments(new SpatialElementBoundaryOptions()).Count > 0)
                 {
-                    return BoundCondition.Redundant;
+                    return BoundCondition.Overlapping;
                 }
                 else
                 {
                     return BoundCondition.NotEnclosed;
                 }
             }
+            else if (AreaVolumeSettings.GetAreaVolumeSettings(elem.Document).ComputeVolumes)
+            {
+                if (elem is Room room && room.Volume == 0
```

Comment:
Wouldn't it be leaving some tolerance, e.g. `room.Volume < 1e-3` or similar?

Reply:
Actually, the volume will always be exactly 0 when the spatial element has no geometry, so I think this is safe enough. This is similar to a space that has been unplaced (deleted from the plan view) which has an area of exactly 0.